pkg/controller/restore_controller_test.go

```diff
@@ -182,8 +182,9 @@ func TestProcessRestore(t *testing.T) {
 			name:                        "restore with non-existent backup name fails",
 			restore:                     arktest.NewTestRestore("foo", "bar", api.RestorePhaseNew).WithBackup("backup-1").WithIncludedNamespace("ns-1").Restore,
 			expectedErr:                 false,
-			expectedPhase:               string(api.RestorePhaseInProgress),
 			expectedRestoreErrors:       1,
```

Comment:
You can remove this - it's actually 0, and the test logic short-circuits before evaluating this value

Reply:
Done